micrometer-core/src/main/java/io/micrometer/core/instrument/AbstractTimer.java

```diff
@@ -82,27 +82,32 @@ protected AbstractTimer(Id id, Clock clock, DistributionStatisticConfig distribu
      */
     protected AbstractTimer(Id id, Clock clock, DistributionStatisticConfig distributionStatisticConfig,
             PauseDetector pauseDetector, TimeUnit baseTimeUnit, boolean supportsAggregablePercentiles) {
+        this(id, clock, pauseDetector, baseTimeUnit,
+                defaultHistogram(clock, distributionStatisticConfig, supportsAggregablePercentiles));
+    }
+
+    protected AbstractTimer(Id id, Clock clock, PauseDetector pauseDetector, TimeUnit baseTimeUnit,
+            @Nullable Histogram histogram) {
```

Comment:
Any reason we should accept `null` here? Do we already have a use case for that?

Reply:
Not really. I guess, we can always pass NoopHistogram.INSTANCE if we don't need histogram and that is what this does unltimately.